### ISSUE TITLE
Term Request: Polychromatic pre-Descemet corneal opacities

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11749]
**Preferred term label:**
Polychromatic pre-Descemet corneal opacities

**Synonyms**
* Polychromatic punctiform pre-Descemet deposits
* Polychromatic pre-Descemet flecks

**Definition (free text, please give PubMed ID)**
Uniform, polychromatic, punctiform opacities situated exclusively within the deep posterior corneal stroma immediately anterior to Descemet membrane in an otherwise clear cornea, distinguishable by their polychromatic nature and larger size from the fine, dust-like lesions of cornea farinata (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Opacification of the corneal stroma (HP:0007759)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Punctiform and polychromatic pre-Descemet corneal dystrophy (PPPCD) (OMIM:619871, ORPHA:293462)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
